sonarapp/Program.cs

```diff
@@ -16,16 +16,9 @@ static void Main(string[] args)
 
             Boolean whatif = (new verify()).verifywhat("whatif");
 
-            Boolean whatiffff = (new verify()).verifywhat("whatififff");
-
-            Boolean whatifsssfff = (new verify()).verifywhat("whatififfsssf");
-
-            Boolean whatifsssfffjenkin = (new verify()).verifywhat("whatifsssfffjenkin");
-
-            Boolean whatifsssfffjenkinpr = (new verify()).verifywhat("whatifsssfffjenkinpr");
-
-            Boolean whatifsssfffjenkinprb = (new verify()).verifywhat("whatifsssfffjenkinprbb");
+            verify v= new verify();
 
+            Boolean verifiyResult = v.verifywhat(string.Empty);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'verifiyResult' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=csharpsquid%3AS1481)